src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Microsoft.AspNetCore.Razor.Test.Common.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- To generate baselines, run tests with /p:GenerateBaselines=true -->
     <DefineConstants Condition="'$(GenerateBaselines)'=='true'">$(DefineConstants);GENERATE_BASELINES</DefineConstants>
     <DefineConstants>$(DefineConstants);__RemoveThisBitTo__GENERATE_BASELINES</DefineConstants>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
Why do we need `net461` here? Do we still have any code in this repo that's net461?

Reply:
See https://github.com/aspnet/AspNetCore-Tooling/pull/187#discussion_r255363653. As part of `TestCompilation`, we attempt to compile against assemblies already in the AppDomain when available. With the new SDK, this no longer works with .NETCoreApp targeting compilation tests (I don't think this is a supported scenario - you get a mix of runtime and ref assemblies that aren't supposed to be referenced simultaneously). However, it is seemingly required for desktop projects. I'll investigate that later, but didn't want to block this PR in the meanwhile. The affected tests were the compilation tests for Mvc.Extensions.V1 and V2, so definitely unrelated to any change being made here.

Reply:
okey dokey. I don't think it's worth burning a lot of time on. I was just really happy to get rid of this cross-targeting.